apache/superset
# Extract widget javascript to separate files.

This is the first step towards a cleaner abstraction for defining widgets.

Test plan
- Run `panoramix/bin/panoramix load_examples`
- Visit http://localhost:8088/panoramix/dashboard/1 and expect all widgets to load
- Visit "Trends" slice and check/uncheck show legend and reload the widget
